x-pack/plugins/cross_cluster_replication/public/app/services/follower_index_default_settings.js

```diff
@@ -28,3 +28,7 @@ export const getSettingDefault = (name) => {
 export const isSettingDefault = (name, value) => {
   return getSettingDefault(name) === value;
 };
+
+export const areAllSettingsDefault = (settings) => {
```

Comment:
You and I are _kind of_ duplicating each other's logic: https://github.com/elastic/kibana/pull/29543/files#diff-09a600da2112444503d4078ae82502d4R246. But I should probably just move the stuff in my PR into services and group it with this file into a module.

Reply:
I noticed that too and figured we could just consolidate after both PRs are merged 🙂